.gitlab/ci/publish.gitlab-ci.yml

```diff
@@ -7,14 +7,25 @@ publish:
   before_script:
     - pip install poetry
   script:
-    # grab the built package
+    # Add the release marker: a file whose presence indicates that this copy of Meltano was
+    # obtained from a Python package index, rather than directly from source. This marker is used
+    # to improve Meltano's telemetry.
+    - pushd /meltano/dist
+    - ARCHIVE="$(sh -c 'echo "$1"' sh meltano*.tar.gz)"
+    - DIR="$(basename $ARCHIVE .tar.gz)"
+    - tar -xvzf "$ARCHIVE"
+    - touch "$DIR/src/meltano/core/tracking/.release_marker"
```

Comment:
Python packaging is weird sometimes about what actually ships and what doesn't. Can we double check this file gets bundled and installed with the operation? (Either in docs or through a 'real' test.)

Also - and I just realized this now - I think this breaks if a user does `pipx install git+https://github.com/meltano/meltano.git`

For now, I think this can be a bug we would live with, but perhaps we open an issue to track/prioritize in a future PR. 

Wdyt?

Reply:
👍 I'll log an issue, im not as well versed with python packaging as y'all so gonna defer to y'all on how realiable this is.

Reply:
https://github.com/meltano/meltano/issues/6021